php/php.dbgp/src/org/netbeans/modules/php/dbgp/ui/DbgpMethodBreakpointPanel.java

```diff
@@ -106,13 +104,10 @@ private void initComponents() {
             layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
             .addGroup(layout.createSequentialGroup()
                 .addContainerGap()
-                .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.TRAILING)
-                    .addGroup(layout.createSequentialGroup()
-                        .addComponent(myStopOnLbl)
-                        .addGap(31, 31, 31))
-                    .addGroup(layout.createSequentialGroup()
-                        .addComponent(myMethodLbl)
-                        .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED)))
+                .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
```

Comment:
Just a note - is the UI (layout) OK? Please, verify it. Thank you.

Reply:
Yes, it's changed because I fixed alignment of "Stop On" label.

This is dialog with this change:
![netbeans-5228-after](https://user-images.githubusercontent.com/4249184/104837190-2f3f4d00-58b3-11eb-8027-929a5f3c462e.png)

Reply:
Great, thank you for letting me know.